From 1cd93b87665d78db7f1953d23c0836d87000c183 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Fri, 10 Jun 2022 15:16:27 +0800
Subject: [PATCH 145/274] MLK-25940-1 arm64: dts: imx93-11x11-evk: add
 eeti,exc80h60 touch support

The LVDS panel has one eeti,exc80h60 touch, so enable it.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Reviewed-by: Jun Li <jun.li@nxp.com>
---
 .../freescale/imx93-11x11-evk-boe-wxga-lvds-panel.dts | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/imx93-11x11-evk-boe-wxga-lvds-panel.dts b/arch/arm64/boot/dts/freescale/imx93-11x11-evk-boe-wxga-lvds-panel.dts
index 9a84c4983..1a8df881c 100644
--- a/arch/arm64/boot/dts/freescale/imx93-11x11-evk-boe-wxga-lvds-panel.dts
+++ b/arch/arm64/boot/dts/freescale/imx93-11x11-evk-boe-wxga-lvds-panel.dts
@@ -74,3 +74,14 @@ lvds_out: endpoint {
 &ldb_phy {
 	status = "okay";
 };
+
+&lpi2c1 {
+	exc80h60: touch@2a {
+		compatible = "eeti,exc80h60";
+		reg = <0x2a>;
+		interrupt-parent = <&pcal6524>;
+		interrupts = <8 IRQ_TYPE_EDGE_FALLING>;
+		reset-gpios = <&pcal6524 17 GPIO_ACTIVE_HIGH>;
+		status = "okay";
+	};
+};
-- 
2.25.1

